lib/cmds/space_cmds/alias_cmds/update.js

```diff
@@ -0,0 +1,58 @@
+import * as logging from '../../../utils/log'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { createManagementClient } from '../../../utils/contentful-clients'
+
+export const command = 'update'
+
+export const desc = 'Update an environment alias'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space alias update --alias-id master --environment-id staging')
+    .option('alias-id', {
+      alias: 'a',
+      describe: 'Id of the alias to create',
+      demandOption: true
+    })
+    .option('environment-id', {
+      alias: 'e',
+      describe: 'ID of the environment to alias to',
+      type: 'string'
+    })
+    .option('space-id', {
+      alias: 's',
+      describe: 'ID of the space that the alias will belong to',
+      type: 'string'
+    })
+    .option('management-token', {
+      alias: 'mt',
+      describe: 'Contentful management API token',
+      type: 'string'
+    })
+    .epilog([
+      'See more at:',
+      'Environment ALiases is an [ALPHA] feature',
+      'https://github.com/contentful/contentful-cli/tree/master/docs/space/alias/update',
+      'Copyright 2018 Contentful, this is a BETA release'
+    ].join('\n'))
+}
+
+export async function environmentAliasUpdate ({ context, aliasId, environmentId }) {
```

Comment:
do we also need to destructure the spaceId here if it's passed in by the user? (same q as above)

Reply:
the middleware will take care of that and make the space available through `context.activeSpaceId`

Reply:
hm then the middleware also takes care of `environmentId` right?